a/apache-freemarker/build_info.json

```diff
@@ -1,17 +1,21 @@
 {
-  "maintainer": "IBM-Currency-Helper",
+  "maintainer": "AbhiIBM",
   "package_name": "apache-freemarker",
   "github_url": "https://github.com/apache/freemarker",
   "version": "v2.3.32",
   "default_branch": "2.3-gae",
   "package_dir": "a/apache-freemarker/",
   "docker_cmd": "docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
-  "build_script": "apache-freemarker_v2.3.31_ubi_8.5.sh",
+  "build_script": "apache-freemarker_ubi_9.3.sh",
   "docker_build": true,
   "validate_build_script": true,
   "use_non_root_user": false,
-  "v2.3.*": {
+  "v2.*.*": {
+    "dir": "ubi9.3",
+    "build_script": "apache-freemarker_ubi_9.3.sh"
+  },
+  "v2.3.31": {
     "dir": "ubi_8.5",
-    "build_script": "apache-freemarker_v2.3.31_ubi_8.5.sh"
+    "build_script": "apache-freemarker_v2.3.31_ubi_9.3.sh"
```

Comment:
@AbhiIBM have you added this script  apache-freemarker_v2.3.31_ubi_9.3.sh", i dont see there. if new scrip is working for old version we can use same there also. Take a look again

Reply:
Removed the script name typo and the new script will work for older versions also so adding new script only.